commit efd89a2204d4086de48673ff7e4d5f537e32e0a5
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Sep 23 16:36:06 2010 -0700

    Staging: brcm80211: remove driver specific -W options
    
    A single driver shouldn't be overriding the kernel-wide -W options.
    This removes them from the Makefile.
    
    Bonus is that the code at least now will build on a 64bit platform.
    Problem is that both drivers can't be built at the same time right now
    or bad things happen when linking.
    
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Cc: jason <jason@lakedaemon.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/Makefile
+++ b/drivers/staging/brcm80211/Makefile
@@ -27,11 +27,7 @@ ccflags-y :=					\
 	-Idrivers/staging/brcm80211/sys		\
 	-Idrivers/staging/brcm80211/phy		\
 	-Idrivers/staging/brcm80211/util	\
-	-Idrivers/staging/brcm80211/include	\
-	-Wall					\
-	-Wstrict-prototypes			\
-	-Werror					\
-	-Wmissing-prototypes
+	-Idrivers/staging/brcm80211/include
 
 PCI_CFLAGS := -DWLC_LOW
 
--- a/drivers/staging/brcm80211/brcmfmac/Makefile
+++ b/drivers/staging/brcm80211/brcmfmac/Makefile
@@ -36,9 +36,6 @@ ccflags-y :=			\
 	-DDHD_GPL		\
 	-DBCMLXSDMMC		\
 	-DBCMPLATFORM_BUS	\
-	-Wall			\
-	-Wstrict-prototypes	\
-	-Werror			\
 	-DOEM_CHROMIUMOS	\
 	-DEMBEDDED_PLATFORM	\
 	-DARP_OFFLOAD_SUPPORT	\
